aunt-leahs-app/src/components/header/header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SubText from './SubText.jsx';
+import constants from '../../constants.jsx';
+import '../../styles.css';
+
+const headerText = constants.HEADER.MAIN_HEADER;
+
+const Head = (props) => {
+    /*takes in 'page' as props. Refer to ./components/constants.jsx for props.page values */
+    return(
+        <div className='headerBox'>
+                <img className="headerImage header"  alt="Aunt Leah's Logo" src={require('../../AuntLeahsTrees.png')}/>
+                <h1 className="headerMain header">{headerText.headTwo}</h1>
```

Comment:
I think this should be a div instead of a h1, then any styling would come from the class

Reply:
noted.